MEMBER_TRAVEL_FUND.md

```diff
@@ -10,40 +10,72 @@ and the Node.js Foundation projects.
 
 ## Process
 
-Any member can apply for travel funds. Each request may take up to a month
-for the [Admin members](https://github.com/nodejs/admin#governance-and-current-members)
-to approve or decline.
+Any member can apply for travel funds.
 
 ### Request
-* Open a pull request against this repository which adds an entry to the table at the bottom of this file.
+
+* Open a pull request against this repository which adds an entry to the table
+  at the bottom of this file.
  * Include the name of the event you plan to attend.
  * Include the location of the event and where you will be traveling from.
  * Include the presentation you intend to give, if applicable.
  * Include the size of the stipend you wish to receive.
-  * Reimbursement stipends are expected to vary with travel distance.
-  * In the PR description, tag the group relevant to the request: @nodejs/tsc or @nodejs/community-committee.  
-  If unsure, just pick one, and that group will tag with the other group if they believe it is more appropriate.
-* Once the final amount spent is known, update the table again with that information.
+   * Reimbursement stipends are expected to vary with travel distance.
+   * In the PR description, mention `@nodejs/tsc` and
+     `@nodejs/community-committee`.  
+* Once the final amount spent is known, update the table again with that
+  information.
 
-### Reimbursement
+### Approval
+
+A request is approved when all of the following criteria are met:
+
+* The pull request has been open for at least 72 hours.
+* The request has approval from at least two members of @nodejs/tsc and two
+  members of @nodejs/community-committee. (People who are members of both
+  committees may only be counted for one or the other. Therefore, any request
+  will always require at least four approvals.)
+* No members of either @nodejs/tsc or @nodejs/community-committee have objected
+  to the request.
 
-Once the request has been approved, provide receipts as attachments in an email stating your name, the participation covered, and the total approved for reimbursement. Send to ap@linuxfoundation.org and cc mrhinkle@linuxfoundation.org with subject `Node.js Member Travel Fund`. Due to privacy, the Individual Members list is not public. This team will verify the requester is on the Individual Members list before funds are dispersed. This dispersement is generally processed within 30 days. The community manager or a member of the Node.js Foundation team within the Linux Foundation will report back amounts consumed from the travel allocation on a monthly basis to the Chairs.
+If a request is blocked by an objection from a member of @nodejs/tsc or
+@nodejs/community-committee, then the only way for the request to be approved is
+for each committee to decide to approve the request using whatever
```

Comment:
I would change `for each committee` to `for the blocking committee(s)`, since I don't think a committee that is not blocking should have to go through a full voting process.

Reply:
I think both committees should. Blocking is so unusual (never happened, as far as I know) that whatever the objections might be, they would likely warrant scrutiny from both committees. This also discourages people from blocking for relatively minor/frivolous reasons.

Reply:
Should both committees need to resolve? Shouldn't only the committee with the objecting member resolve their contention?

Reply:
@MylesBorins @bnb I feel like an objection should be taken with utmost seriousness. We've never had one, so if we ever do, it warrants pause for everyone. But it's hard to argue that what you're saying isn't the most straightforward and sensible. So I relent! I'm going with what you suggest. Please consider piling on with approvals. Thanks.